docs/contents/releases/1-19/1/index.md

```diff
@@ -2,7 +2,7 @@
 
 Highlights of the Kubernetes 1.19 release include Ingress API and Pod Topology
 Spread reaching stable status, EndpointSlices being enabled by default, and
-immutable Secrets and ConfigMaps. A [change log](CHANGELOG-1.19.1.md) is available
+immutable Secrets and ConfigMaps. A [change log](CHANGELOG-v1-19-eks-1.md) is available
 describing the patches and versions included in this release.
 
 The [EKS-D v1-19-eks-1](https://distro.eks.amazonaws.com/kubernetes-1-19/kubernetes-1-19-eks-1.yaml)
```

Comment:
CRD - Custom Resource Definition
Did we decide to use CRD as Custom `Release` Document/Definition for the purpose of release? 
If speaking in terms of Kubernetes resource, we can go with CRD - Custom Resource Definition. 
If speaking in terms of a file or document, I think EKS-D v1-19-eks-1 release manifest would sound better.

Reply:
@micahhausler ?

Reply:
Yea, I'm ok with the `v1-19-eks-1 release manifest` text